src/components/booksForm/booksForm.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable arrow-body-style */
+
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { add } from '../../containers/actions/index';
+
+const BookForm = ({ addBook }) => {
+  const handleForm = (event) => {
+    const title = event.target.title.value;
+    const categorie = event.target.categorie.value;
+    addBook({
+      title,
+      categorie,
+    });
+    event.preventDefault();
+  };
```

Comment:
## Implement the missing book id section 
- [ ] Our `book` component should represent a single row of books with `id`, `title` and `category` [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/RepresentARowInBooksTable.png). We are currently missing the `id` part, let's implement it here. We can have this by generating a random number to act as our `id` whenever a book gets created. that's one way to do it.